source/install/prod-ubuntu-16.04.rst

```diff
@@ -0,0 +1,489 @@
+- `Set up Mattermost Server`_
+- `Set up NGINX Server`_
+- `Post-Installation Mattermost Configuration`_
+
+
+Install Ubuntu Server (x64) 16.04 LTS
+-------------------------------------
+
+1. Set up 3 machines with Ubuntu 16.04 with 2GB of RAM or more. The
+   servers will be used for the Proxy, Mattermost (must be
+   x64), and Database.
+
+   -  **Optional:** You can also use a **1 machine setup** (Proxy, Mattermost and Database on one machine) or a **2 machine setup** (Proxy and Mattermost on one machine, Database on another) depending on your data center standards.
+
+2. Make sure the system is up to date with the most recent security
+   patches.
+
+   ::
+
+       sudo apt update
+       sudo apt upgrade
+
+
+Set up Database Server
+----------------------
+
+1.  For the purposes of this guide we will assume this server has an IP
+    address of ``10.10.10.1``.
+2.  Install PostgreSQL 9.5+::
+
+        sudo apt-get install postgresql postgresql-contrib
+
+3.  PostgreSQL created a user account called ``postgres``. You will need
+    to log into that account with::
+
+        sudo -i -u postgres
+
+4.  You can get a PostgreSQL prompt by typing::
+
+        psql
+
+5.  Create the Mattermost database by typing::
+
+        postgres=# CREATE DATABASE mattermost;
+
+6.  Create the Mattermost user by typing::
+
+        postgres=# CREATE USER mmuser WITH PASSWORD 'mmuser_password';
+
+7.  Grant the user access to the Mattermost database by typing::
+
+        postgres=# GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;
+
+8.  You can exit out of PostgreSQL by typing::
+
+        postgres=# \q
+
+9.  You can exit the postgres account by typing::
+
+        exit
+
+10. Allow Postgres to listen on all assigned IP Addresses
+
+    -  Edit the config file::
+
+           sudo vi /etc/postgresql/9.5/main/postgresql.conf
+
+    -  Uncomment ``listen_addresses`` and change ``localhost`` to ``*``
+
+11. Alter ``pg_hba.conf`` to allow the mattermost server to talk to the
+    postgres database
+
+    -  Edit the config file::
+
+        sudo vi /etc/postgresql/9.5/main/pg_hba.conf
+
+    -  Add the following line to the ``IPv4 local connections``::
+
+        host all all 10.10.10.2/32 md5
+
+12. Reload Postgres database::
+
+        sudo systemctl reload postgresql.service
+
+    Check with ``netstat`` command to see postgresql actually running on given ip and port::
+
+        sudo netstat -anp | grep 5432
+
+    Try restarting the postgresql service if reload won't work::
+
+        sudo systemctl restart postgresql.service
+
+13. Attempt to connect with the new created user to verify everything
+    looks good::
+
+        psql --host=10.10.10.1 --dbname=mattermost --username=mmuser --password
+        mattermost=> \q
+
+
+Set up Mattermost Server
+------------------------
+
+1. For the purposes of this guide we will assume this server has an IP
+   address of ``10.10.10.2``
+2. For the sake of making this guide simple we located the files at
```

Comment:
We should be consistent with this [PR](https://github.com/mattermost/docs/pull/573) and use `mattermost` user and group